### PR TITLE
count Rally users and extensions from auth db hourly and log

### PR DESCRIPTION
**Pull request by rhelmer**
Review expectations: This just changes this from an HTTP-triggered function to run on a schedule, I will test this on stage before deploying. OK to just look, it should be possible to write automated tests against the emulator but this is still in flux so I haven't tried yet.

### CODE REVIEW

**Comment by rhelmer** [comment]
I pushed the latest version to stage to test.

**Comment by rhelmer** [comment]
This is working overall, but deployment to stage (which shouldn't be happening in PRs, see https://github.com/mozilla-rally/rally/issues/127) is failing.

The service account is going to need access to the appropriate GCP service to be able to create scheduled tasks, and we'll need to delete the function that I pushed with my account.

**Review by aaga — APPROVED** [review]
looks good to me.

is this going to trigger that "confirm via CLI" thing again that happened when we added "retry on failure" to our functions? Could that be why deploy is failing?

**Comment by rhelmer** [comment]
> looks good to me.
> 
> 
> 
> is this going to trigger that "confirm via CLI" thing again that happened when we added "retry on failure" to our functions?

Yep that's exactly why, we need to run it from the cli and manually accept I suppose

**Comment by rhelmer** [comment]
I'll do a manual deployment before we land this, so we don't break CI on `main`.

File: web-platform/functions/src/index.ts [hunk 1]
```diff
@@ -332,24 +332,21 @@ async function getSchemaNamespaceforStudy(studyID: string) {
...
 
 /**
- * Lists users.
- *
- * @param {!express:Request} req HTTP request context.
- * @param {!express:Response} res HTTP response context.
+ * Counts Rally users and stores aggregates in cloud logs. Runs on a schedule.
  */
-export const countRallyUsers = functions.https.onRequest(async (req, res) => {
+export const countRallyUsers = functions.pubsub.schedule('every hour').onRun(async (_context: EventContext) => {
```
Comment (thomik-corp) [inline web-platform/functions/src/index.ts:337]: Minor "" instead of '' as prettier will barf.

**Review by thomik-corp — APPROVED** [review]

**Pull request merged**